src/sentry/models/team.py

```diff
@@ -160,6 +160,7 @@ class Team(Model, SnowflakeIdMixin):
     idp_provisioned = models.BooleanField(default=False)
     date_added = models.DateTimeField(default=timezone.now, null=True)
     org_role = models.CharField(max_length=32, null=True)
+    through_project_creation = models.BooleanField(default=False)
```

Comment:
you should generate a migration for this

Reply:
oh just kidding your migration is in another PR. you can remove this line from here then